CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ De change uit GitHub issue [#1472](https://github.com/dso-toolkit/dso-toolkit/is
 * **css:** Vertical margin lijsten en Definition List optimalisatie ([#1472](https://github.com/dso-toolkit/dso-toolkit/issues/1472))
 
 ### Added
+* **core:** Afbeelding groot kunnen weergeven ([#1444](https://github.com/dso-toolkit/dso-toolkit/issues/1444))
```

Comment:
Bij mergen moet deze CHANGELOG entry verplaatst worden naar de UNRELEASED sectie

Reply:
hoezo dat dan?

Reply:
Omdat ie nu in de release van 34.2.1 staat